src/enums/components/connection.enum.ts

```diff
@@ -221,6 +223,11 @@ export const IntegrationsMap: Record<Integrations, IntegrationSelectOption> = {
 		label: "Microsoft Teams",
 		value: Integrations.microsoft_teams,
 	},
+	kubernetes: {
```

Comment:
@itay – Should this be hidden in certain environments (production, or maybe for specific named customers) using a [feature flag](https://github.com/autokitteh/web-platform/blob/8da69a48d48aedce9ed6e5d720f40e4352747c45/src/constants/featureFlags.constants.ts#L7), or should we display it to everyone?
Since it doesn’t require special credentials or configuration unlike Zoom, Salesforce, or Linear, I think it makes sense to show it to everyone. Do you agree?